docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.5"
+
+services:
+  db:
+    image: postgres:12.1
+    container_name: go_google_scraper_db
+    environment:
+      - POSTGRES_DB=go_google_scraper_development
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+
+  app:
```

Comment:
Minor comment but prefer to organize services in alphabetical order and also usually placing the app at the top:

```
services:
  app:

  db:
```

Reply:
Fixed in 8d00ee48d876299bc15e5d888bb7915901b6da3d